java/ql/src/semmle/code/java/frameworks/apache/Lang.qll

```diff
@@ -1,27 +1,58 @@
 /** Definitions related to the Apache Commons Lang library. */
 
 import java
+private import semmle.code.java.dataflow.FlowSteps
```

Comment:
This needs bidirectional import to work, i.e. you need to add `private import java/ql/src/semmle/code/java/frameworks/apache/Lang.qll` in `FlowSteps.qll`